docs/content/2.guide/4.custom-paths.md

```diff
@@ -41,10 +41,6 @@ Customized route paths **must start with a `/`** and **not include the locale pr
 You can now use the `localePath` function or the `<NuxtLinkLocale>` component but be sure to use named routes. For example route `/services/advanced` should be `services–advanced`:
 
 ```vue
-<script setup>
-const { t } = useI18n()
-</script>
-
 <template>
   <NuxtLinkLocale to="about">{{ $t('about') }}</NuxtLinkLocale>
   <NuxtLinkLocale to="services–advanced">{{ $t('advanced') }}</NuxtLinkLocale>
```

Comment:
This should not be removed.
The template is a bad example. It should be changed to use the composable `t` instead of the global `$t` in this example case.